### PR TITLE
Add tt run and tt instances

### LINKED ISSUE #3001: tt run
**Epic: [Document TT CLI](https://github.com/tarantool/doc/issues/2553)**
Related dev issue: https://github.com/tarantool/tt/issues/29

Product: Tarantool CLI
Since: 2.10
Audience/target: dev, admin
Root document: TBA
SME: @ LeonidVas

### PR DESCRIPTION
**Pull request by p7nov**
Resolves #3001 #3318

### CODE REVIEW

File: doc/reference/tooling/tt_cli/check.rst [hunk 1]
```diff
@@ -20,14 +20,14 @@ specified in the :ref:`tt configuration file <tt-config_file_app>`. ``APP_FILE``
...
-*   Check the syntax of the ``app.lua`` file from the ``instances_available`` directory:
+*   Check the syntax of the ``app.lua`` file from the ``instances_enabled`` directory:
 
     ..  code-block:: bash
 
         tt check app
 
 
-*   Check the syntax of the ``init.lua`` file from the ``instance1/`` directory inside ``instances_available``:
+*   Check the syntax of the ``init.lua`` file from the ``instance1/`` directory inside ``instances_enabled``:
```
Comment (psergee) [inline doc/reference/tooling/tt_cli/check.rst:30]: Looks like it must be "from the ``app/`` directory"

File: doc/reference/tooling/tt_cli/run.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
+.. _tt-run:
+
+Running code in a Tarantool instance
+====================================
+
+..  code-block:: bash
+
+    tt run SCRIPT|-e EXPR [flags]
```
Comment (psergee) [inline doc/reference/tooling/tt_cli/run.rst]: `tt run` can be invoked without script/expression. So, these args must be in square brackets. `tt run` without arguments invokes `tarantool` interpreter from current `tt` environment.

File: doc/reference/tooling/tt_cli/check.rst (hunk 1 above)
Comment (p7nov) [reply doc/reference/tooling/tt_cli/check.rst:30]: Nope, I rechecked this.
The call `tt check app` above this line is the previous example, if you're talking about it.
See [here](https://docs.d.tarantool.io/en/doc/tt-run/reference/tooling/tt_cli/check/)
image

File: doc/reference/tooling/tt_cli/instances.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    tt instances
+
+``tt instances`` shows the list of enabled applications.
+
+.. note::
+
+    *Enabled* applications are applications that are stored inside the ``instances_enabled``
+    directory specified in the :ref:`tt configuration file <tt-config_file_app>`.
+    They can be either be running or not. To check if an application is running,
```
Comment (andreyaksenov) [inline doc/reference/tooling/tt_cli/instances.rst]: be either be -> be either

File: doc/reference/tooling/tt_cli/run.rst [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+``tt run`` executes arbitrary Lua code in a Tarantool instance. The code can be
+provided either in a Lua file, or in a string passed after the ``-e``/``--evaluate``
+flag. When called without arguments or flags, ``tt run`` opens the Tarantool console.
+
+If libraries are required for execution, pass their names after the ``-l``/``--library``
+flag.
+
+By default, a Tarantool instance started by ``tt run`` shuts down after the code
+execution completes. To leave this instance running and continue working in its
```
Comment (andreyaksenov) [inline doc/reference/tooling/tt_cli/run.rst:49]: > after the code execution completes 

It looks like 'the' should be removed.

File: doc/reference/tooling/tt_cli/check.rst [hunk 5]
```diff
@@ -10,7 +10,7 @@ Checking an application file
 Details
 -------
 
-``tt`` searches for ``APP_FILE`` inside the ``instances_available`` directory
+``tt`` searches for ``APP_FILE`` inside the ``instances_enabled`` directory
```
Comment (andreyaksenov) [inline doc/reference/tooling/tt_cli/check.rst:13]: To double-check: do you mean '``tt`` searches' or '``tt check`` searches' here?

**Review by andreyaksenov — APPROVED** [review]
Looks great, just a few comments.

File: doc/reference/tooling/tt_cli/check.rst (hunk 5 above)
Comment (p7nov) [reply doc/reference/tooling/tt_cli/check.rst:13]: This sentence describes the general principle that is used in several tt commands, so I think ``tt searches`` is fine.

File: doc/reference/tooling/tt_cli/instances.rst (hunk 3 above)
Comment (p7nov) [reply doc/reference/tooling/tt_cli/instances.rst]: fxd

File: doc/reference/tooling/tt_cli/run.rst (hunk 4 above)
Comment (p7nov) [reply doc/reference/tooling/tt_cli/run.rst:49]: agree, fxd

File: doc/reference/tooling/tt_cli/run.rst (hunk 2 above)
Comment (p7nov) [reply doc/reference/tooling/tt_cli/run.rst]: thanks, added the brackets and a sentence explaining this point.

**Pull request merged**